.github/workflows/github-pages.yml

```diff
@@ -0,0 +1,19 @@
+name: Build and deploy to GitHub Pages
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+  push:
+    branches:
+      - develop
```

Comment:
I don't see PRs being tested? Travis was doing that IIRC.

Reply:
Ah yes, I'll have a look

Reply:
Preferably it would be nice if PRs could deploy to surge.sh to preview how the website would look like